tests/test_crud.py

```diff
@@ -7,48 +7,8 @@
 import pytest
 import yaml
 
-MODULES = [
-    'activation_key',
-    'architecture',
-    'auth_source_ldap',
-    'bookmark',
-    'compute_attribute',
-    'compute_profile',
-    'compute_resource',
-    'content_credential',
-    'content_view',
-    'content_view_filter',
-    'domain',
-    'environment',
-    'config_group',
-    'global_parameter',
-    'host',
-    'hostgroup',
-    'host_collection',
-    'installation_medium',
-    'job_template',
-    'katello_manifest',
-    'lifecycle_environment',
-    'location',
-    'operating_system',
-    'organization',
-    'os_default_template',
-    'product',
-    'provisioning_template',
-    'ptable',
-    'redhat_manifest',
-    'repository',
-    'repository_set',
-    'repository_sync',
-    'role',
-    'search_facts',
-    'setting',
-    'snapshot',
-    'subnet',
-    'sync_plan',
-    'upload',
-    'user',
-]
+from .conftest import TESTED_MODULES
```

Comment:
You could make this a fixture and use it again elsewhere. https://github.com/theforeman/obal/blob/2f21ce6999b337bb1efd1f8c6cc12287242510d4/tests/conftest.py#L5-L7 defines a shared fixture. https://github.com/theforeman/obal/blob/2f21ce6999b337bb1efd1f8c6cc12287242510d4/tests/test_playbooks.py#L17-L19 parametrizes on a list.

Reply:
But I also need is a plain list in https://github.com/theforeman/foreman-ansible-modules/blob/229cf0a1a573d998dddff3c7a24d6704cf7d1635/tests/test_module_state.py#L33-L37

would that still work?

Reply:
I'd have a plain list and then build fixture from that.

From https://gist.github.com/ekohl/bcb4b815153f491d08d551ed4ed98ad9#file-test_pages-py-L67-L69

```
PAGES = [
  # ...
]

@pytest.fixture(params=PAGES)
def url(request, base_url):
    return base_url + request.param
```

Reply:
What's the benefit compared to directly using `@pytest.mark.parametrize('module', TEST_PLAYBOOKS)`?

Reply:
If it's one occurrence then probably little to none.